app/utils.py

```diff
@@ -0,0 +1,16 @@
+import typing as t
+
+import yaml
+from models import AdditionalConfig
+
+
+def load_yaml() -> dict[str, t.Any]:
+    with open("config.yaml") as stream:
+        try:
+            admin_config = yaml.safe_load(stream)
+            return AdditionalConfig(**admin_config)
+        except yaml.YAMLError as exc:
+            raise exc
```

Comment:
Sorry to be a pain, but can we change the return type to `AdditionalConfig`. We may we well not bother catching the exception if we're just going to re-raise it. Unless we print something out to explain the error to the user.

```python
def load_yaml() -> AdditionalConfig:
    with open("config.yaml") as file:
        admin_config = yaml.safe_load(file)
        return AdditionalConfig(**admin_config)
```

Reply:
No problem, I'll change that since we already mount the correct path to `yaml` file in the container.